<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="aTab">
         <attribute name="title">
          <string>Auto Clicker</string>
         </attribute>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>461</width>
            <height>51</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>71</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="plainTextEdit_2">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>10</y>
             <width>71</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="plainTextEdit_3">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>10</y>
             <width>71</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="plainTextEdit_4">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>10</y>
             <width>71</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>10</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Hours</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>10</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Mins</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>10</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Secs</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>10</y>
             <width>21</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>MS</string>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>1</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Time Interval</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QFrame" name="frame_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>461</width>
            <height>61</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Click Loop</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>30</y>
             <width>101</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Until Stopped</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>10</y>
             <width>89</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Loop</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinBox">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>10</y>
             <width>61</width>
             <height>21</height>
            </rect>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>10</y>
             <width>31</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>times</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>10</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Mouse Button</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>30</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Click Choice</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>10</y>
             <width>69</width>
             <height>21</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Left</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Right</string>
            </property>
           </item>
          </widget>
          <widget class="QComboBox" name="comboBox_2">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>32</y>
             <width>69</width>
             <height>20</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Single</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Double</string>
            </property>
           </item>
          </widget>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>60</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Options</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QFrame" name="frame_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>140</y>
            <width>461</width>
            <height>121</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>20</y>
             <width>151</width>
             <height>61</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>START</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>20</y>
             <width>151</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>STOP</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QKeySequenceEdit" name="keySequenceEdit">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>81</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="keySequence">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
           <property name="maximumSequenceLength">
            <longLong>4</longLong>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>85</y>
             <width>49</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Hotkey</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="bTab">
         <attribute name="title">
          <string>Auto Keyboard Clicker</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="cTab">
         <attribute name="title">
          <string>Macro Automation Player</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Keybinds</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
